src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -69,7 +69,7 @@ class DoorLockClusterFabricDelegate : public chip::FabricTable::Delegate
 };
 static DoorLockClusterFabricDelegate gFabricDelegate;
 
-void emberAfPluginDoorLockOnAutoRelock(chip::EndpointId endpointId);
+void DoorLockOnAutoRelockCallback(chip::EndpointId endpointId);
```

Comment:
```suggestion
static void DoorLockOnAutoRelockCallback(chip::EndpointId endpointId);
```

But actually, is there a reason to not just make this a static member function of DoorLockServer?  Then you won't need the friend declaration either....

Reply:
`AutolockEvent` was private from that context so the way it was placed it needed to be a friend of the door lock class. I previously just renamed the old function but you are right. Now, I changed this to make `DoorLockOnAutoRelockCallback` a static private member of the Door lock class and remove the local declaration. This is a better representation of what the callback is